micr_XPCTzarr/dataset_description.json

```diff
@@ -0,0 +1,14 @@
+{
+	"Name": "micr_XPCTzarr",
+	"Authors": [
+		"Matthieu Chourrout",
+		"David Stansby",
+		"Claire L. Walsh",
+		"Peter D. Lee"
+	],
```

Comment:
@dstansby @Guillaume-Gaisne @ucbpclw
Which authors should I add?

Reply:
Since this is just an example and not a real dataset, I think fine to just leave this as-is?